app/templates/components/versions-menu.hbs

```diff
@@ -0,0 +1,25 @@
+<a class="dropdown-toggle" data-toggle="dropdown" href="#">
+  Dependencies <b class="caret"></b>
+</a>
+
+<ul class="dropdown dropdown-menu">
+  <li class="dropdown-submenu">
+    <a tabindex="-1">Ember.js</a>
+
+    <ul class="dropdown dropdown-menu" >
+      {{#each emberVersions as |version| }}
+        <li><a class="test-set-ember-version" {{action "versionSelected" "ember" version }}>{{version}}</a></li>
```

Comment:
if this uses `{{action versionSelected "ember" version}}` you can remove the `actions` section from `app/components/versions-menu.js` (the `{{action` helper accepts a function, which a closure action is).